### ISSUE TITLE
analysis for the use of the `data` submodule

### ISSUE THREAD

**Issue by eefahy** [sul-dlss/bassi_veratti#12]
Seems like we could possibly manage the EAD XML files for this application within the source code repository. We should investigate the purpose of that to see if it makes sense to maintain those files within one repository.

**Comment by atz** [comment]
Yeah, that is sorta anti-pattern unless the test data is being actively used by more than one repo.  Not really important though.
